web/src/settings_components.ts

```diff
@@ -1203,14 +1203,17 @@ export function save_discard_realm_settings_widget_status_handler($subsection: J
 
 export function save_discard_stream_settings_widget_status_handler(
     $subsection: JQuery,
-    sub: StreamSubscription,
+    sub: StreamSubscription | undefined,
```

Comment:
Under what circumstances can it be `undefined` here? I would have thought this component only existed on the "stream edit" panel, and we by definition need to have a channel data object for such an open channel.

Reply:
It can be undefined when creating a new stream, I can update the commit message.